benefits/serializers.py

```diff
@@ -0,0 +1,8 @@
+from rest_framework import serializers
+from .models import Benefit
+
+
+class BenefitSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Benefit
+        fields = ('id', 'title', 'description', 'category', 'link', 'img_file', 'is_verified')
```

Comment:
- for including all fields use `fields = '__all__'`.
- for excluding fields,  try `exclude=('fields', 'excluded')`.